src/component/variantPage/FunctionalPrediction.tsx

```diff
@@ -63,18 +82,17 @@ class FunctionalPrediction extends React.Component<IFunctionalPredictionProps> {
     }
     public render() {
         const data = this.getData(this.props.variantAnnotation);
-        // Mutation Assessor only available in grch37
-        const shouldShowMutationAssessor =
-            SHOW_MUTATION_ASSESSOR &&
-            this.props.genomeBuild === GENOME_BUILD.GRCh37;
+        const shouldShowAlphaMissense =
+            SHOW_ALPHAMISSENSE &&
+            (this.props.genomeBuild === GENOME_BUILD.GRCh37
+            || this.props.genomeBuild === GENOME_BUILD.GRCh38);
```

Comment:
No need to check the genome build in this case, it will always be either grch37 or grch38, no other options available. Just the SHOW_ALPHAMISSENSE should do the work

Reply:
changed it